xml-parser/src/main/java/net/adoptopenjdk/icedteaweb/xmlparser/ParseException.java

```diff
@@ -49,29 +48,29 @@ public ParseException(String message) {
      */
     public ParseException(String message, Throwable cause) {
         super(getParserSettingsMessage() + message, cause);
-    }  
+    }
 
     public ParseException(Throwable cause) {
         super(getParserSettingsMessage(), cause);
     }
-    
-    
-    static enum UsedParsers {
+
+
+    public enum UsedParsers {
```

Comment:
maybe the enum can be directly extracted in its own java file

Reply:
done